### PR TITLE
feat: MPL2 license removal, cleanup, API fix

### PR DESCRIPTION
**Pull request by simbleau**
In this PR:

### changed

- MPL 2.0 is no longer a license requirement
- The root image viewBox clipping was removed, to be added back at a later time

### fixed

- `vello_svg::render_tree_with` no longer takes a transform parameter. This is to make it consistent with the documentation and `vello_svg::render_tree`.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -71,21 +64,27 @@ pub fn render_tree(scene: &mut Scene, svg: &usvg::Tree) {
...
+    )
 }
 
+// A helper function to render a tree with a given transform.
 fn render_tree_impl<F: FnMut(&mut Scene, &usvg::Node) -> Result<(), E>, E>(
     scene: &mut Scene,
-    svg: &usvg::Tree,
+    tree: &usvg::Tree,
     view_box: &usvg::ViewBox,
     ts: &usvg::Transform,
```
Comment (DJMcNab) [inline src/lib.rs:83]: Is this ever not identity now?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,10 +13,13 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 ### changed
 
 - Updated `usvg` to 0.41
+- MPL 2.0 is no longer a license requirement
+- The root image viewBox clipping was removed, to be added back at a later time
```
Comment (DJMcNab) [inline CHANGELOG.md:17]: Maybe this should be in the "Removed" section?

**Review by DJMcNab — APPROVED** [review]

File: src/lib.rs (hunk 1 above)
Comment (simbleau) [reply src/lib.rs:83]: Yes.

`render_tree_impl` is called from `render_tree`, but it also gets called recursively for group nodes, which pass the parent transform down.

File: CHANGELOG.md (hunk 2 above)
Comment (simbleau) [reply CHANGELOG.md]: Done

**Pull request merged**